pm/sender.go

```diff
@@ -86,14 +86,6 @@ func (s *sender) validateSender(info *SenderInfo) error {
 		return ErrSenderValidation{fmt.Errorf("unable to validate sender: deposit and reserve is set to unlock soon")}
 	}
 
-	if info.Reserve.FundsRemaining.Cmp(big.NewInt(0)) == 0 {
```

Comment:
In light of the comment above, shouldn't we keep these checks but skip them if price is zero?

Reply:
+1

Also if this means sending price everywhere along with sender address like done in the `faceValue` function before, and that leading to unclean code - maybe we can try to see if it makes sense to add `price` as a field in one of the structs (maybe SenderInfo, im not sure?)

Reply:
Fixed in 1964464 - see [this comment](https://github.com/livepeer/go-livepeer/pull/1616#issuecomment-698581224) for more details on the overall changes.